src/Backend/Modules/Slideshows/Layout/Templates/Add.html.twig

```diff
@@ -16,13 +16,15 @@
         </label>
         {% form_field title %} {% form_field_error title %}
       </div>
-      <div class="form-group">
-        <label for="title" class="control-label">
-          {{ 'lbl.Template'|trans|capitalize }}
-          <abbr data-toggle="tooltip" title="{{ 'lbl.RequiredField'|trans|capitalize }}">*</abbr>
-        </label>
-        {% form_field template %} {% form_field_error template %}
-      </div>
+      {% if template %}
```

Comment:
did you test this?
cause if I remember correctly this won't work since it needs to be ddmTemplate